control/control.leanos.xml

```diff
@@ -179,21 +179,21 @@ textdomain="control"
             <archs>x86_64</archs>
           </base_product>
           <base_product>
-            <display_name>SUSE Manager Server 4.1</display_name>
+            <display_name>SUSE Manager Server 4.2</display_name>
             <name>SUSE-Manager-Server</name>
             <version>4.1</version>
```

Comment:
We need also to update the version which is sent to SCC (in the `<version>` tag).

Reply:
Ouch! Of course. Didn't notice the tag :-)

Reply:
Done.